cli/core/docs/cli/cli-architecture.md

```diff
@@ -147,76 +193,16 @@ tanzu cluster list
 tanzu kubernetes cluster list
 ```
 
-List TMC workload clusters:
+List TMC workload clusters using `cluster` plugin associated with `tmc` target:
```

Comment:
If you decide to take my suggestion above, the matching would be here:
"To list TMC workload clusters using the TMC `cluster` plugin which is associated with the `tmc` target:"